src/sv-pipeline/scripts/downstream_analysis_and_filtering/apply_minGQ_filter.py

```diff
@@ -300,7 +300,7 @@ def _get_minGQ(record, minGQ_dict, SVLEN_table, AF_table, SVTYPE_table,
         for key in list(set(EV_table.values())):
             minGQ[key] = int(globalMin)
 
-    return(minGQ)
+    return (minGQ)
```

Comment:
Parenthesis have no effect
```suggestion
    return minGQ
```